config/samples/memcached_v1beta1_memcached.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   name: memcached
 spec:
   replicas: 1
+  cacheSize: 1024
```

Comment:
Is that required? I haven't tried the patch, but I would expect no change in memory behavior without it.

Reply:
I'll remove it, I think I just used this for testing.